From 03d35207d67272ef9d65d49c460747511f41284e Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@protonmail.com>
Date: Sun, 5 May 2024 13:59:59 -0600
Subject: [PATCH 1/1] Disable 'generator' executable

This executable appears to be for assisting the automatic generation of
LSP types and is not of any use to the users of this library.

Signed-off-by: hololeap <hololeap@protonmail.com>
--- a/lsp-types.cabal
+++ b/lsp-types.cabal
@@ -560,6 +560,7 @@ library lsp-types-quickcheck
     , text              >=1       && <2.2
 
 executable generator
+  buildable: False
   hs-source-dirs:     generator
   default-language:   GHC2021
   default-extensions:
-- 
2.43.2

